test/system/700-play.bats

```diff
@@ -549,3 +549,19 @@ EOF
 
     run_podman kube down $yaml_source
 }
+
+@test "podman kube generate filetype" {
+    YAML=$PODMAN_TMPDIR/test.yml
+    run_podman create --pod new:pod1 --security-opt label=level:s0:c1,c2 --security-opt label=filetype:usr_t --name test1 $IMAGE true
+    run_podman kube generate pod1 -f $YAML
+    run cat $YAML
+    is "$output" ".*filetype: usr_t" "Generated YAML file should container filetype usr_t"
```

Comment:
```suggestion
    is "$output" ".*filetype: usr_t" "Generated YAML file should contain filetype usr_t"
```

Reply:
My fingers automatically add the er whenever I type contain ...